services/release-controller-test/admin_02_rbac.yaml

```diff
@@ -0,0 +1,183 @@
+kind: List
+apiVersion: v1
+items:
+- kind: ServiceAccount
+  apiVersion: v1
+  metadata:
+    name: release-controller-test
+    namespace: release-controller-test
+
+# pull access to authenticated users
```

Comment:
Why?

Reply:
I assume that this allows anyone on the cluster the ability to pull from all the imagestreams created in the "release" namespace.  For the production release-controller, that is `ocp`.  For the purpose of our test environment, the "release" namespace is actually **release-controller-test-release**.  I doubt we need this level of access for the test environment, but I took a large majority of this logic directly from `ci-operator/infra/openshift/release-controller/deploy-ocp-controller.yaml`, and Im trying to be as consistent, with the production controller, as possible (for now).